src/charm.py

```diff
@@ -116,13 +117,28 @@ def __init__(self, *args):
             source_type="prometheus",
             source_url=self.external_url,
         )
-
         self.alertmanager_consumer = AlertmanagerConsumer(
             charm=self,
             relation_name="alertmanager",
         )
 
-        # Event handlers
+        self.catalogue = CatalogueConsumer(
+            charm=self,
+            refresh_event=[
+                self.on.prometheus_pebble_ready,
+                self.on["ingress"].relation_joined,
+            ],
+            item=CatalogueItem(
+                name="Prometheus",
+                icon="chart-line-variant",
+                url=self.external_url,
+                description=(
+                    "Prometheus collects, stores and serves metrics as time series data, "
+                    ",alongside optional key-value pairs called labels."
```

Comment:
Wont this render as:
"Prometheus collects, stores and serves metrics as time series data, ,alongside optional key-value pairs called labels."

Reply:
it will! thanks